gradle/libs.versions.toml

```diff
@@ -18,7 +18,10 @@ gtoSupport = "3.11.2-SNAPSHOT"
 jacoco = "0.8.7"
 kotlin = "1.6.21"
 kotlinCoroutines = "1.6.2"
-okhttp3 = "4.10.0"
+
+lottie = "5.2.0"
+okhttp3 = "4.9.3"
+
```

Comment:
as you were addressing this merge conflict you ended up downgrading `okhttp3` to the previous version. Also the extra spaces around these versions should be removed.